ggu/main.c

```diff
@@ -682,6 +682,8 @@ run_ggu(struct net_config *net_conf,
 		ggu_conf->log_ratelimit_interval_ms,
 		ggu_conf->log_ratelimit_burst);
 
+	net_conf->front.total_pkt_burst += ggu_conf->ggu_max_pkt_burst;
```

Comment:
Account it to the **back** interface since the GGU only reads the back interface.